### PR TITLE
Indicate that ENTER can be pressed when fullscreen is enabled and ava…

### PR DESCRIPTION
**Pull request by paxperscientiam**
…ilable

OK, this is the correct followup to issue #129 

ping @GenaBitu

### CODE REVIEW

File: docs/javascripts/imagelightbox.js [hunk 1]
```diff
@@ -158,6 +162,9 @@
                 if (options.caption) {
                     $wrapper.append($captionObject);
                 }
+                if (options.fullscreen) {
```
Comment (rejas) [inline docs/javascripts/imagelightbox.js]: could you add the option to the docs and the fullscreen example?

**Comment by paxperscientiam** [comment]
@rejas what you think now?

**Comment by rejas** [comment]
I still dont see it added to the docs/examples?

**Comment by paxperscientiam** [comment]
@rejas fullscreen option is already described in the docs. Unless, were you thinking that the notice itself should be optional? Because, that's not how it's coded atm.

**Comment by rejas** [comment]
You are right... No need for more docs. But I noticed that you changed thecss file in the docs dir, but not in the src dir... So stuff like @keyframes fadeinout .... is lost and I cannot test it ... Please fix.

**Comment by paxperscientiam** [comment]
No need to rush this. We'll get it right guys. Thanks for help.

**Comment by paxperscientiam** [comment]
@GenaBitu OK, addressed visibility. Now to think about internationalization.

**Comment by GenaBitu** [comment]
Good work.

The navigation is somehow broken - when I press enter, it just adds more and more "dots" - but it does so even on `develop`, so not an issue with this PR :)

**Comment by marekdedic** [comment]
Hi, @paxperscientiam, I am torn on this one - do you think it's worth rebasing onto master or would it be easier to just redo it?

**Comment by paxperscientiam** [comment]
@marekdedic don't rebase if you can avoid it. As for the code, I think it would worth reviewing best practices for shortcuts, especially in light of #912

**Comment by paxperscientiam** [comment]
also, if there's interest. I could recode

**Comment by marekdedic** [comment]
I've been thinking about it a little bit more and maybe we should scrap this PR completely and instead of adding an element saying "You can go fullscreen by pressing Enter", we should just add a fullscreen button?

Currently, to be able to open fullscreen, you need both of the options `enableKeyboard` and `fullscreen` to be `true`. That's kind of stupid and I would be up for changing it - `enableKeyboard` enables the "press Enter for fullscreen" functionality and `fullscreen` would show the fullscreen button.

I am currently adding a unified `ilb-button` class as part of #936, so the fullscreen button could build on that?

Let me know what you think...

**Comment by marekdedic** [comment]
I think this is obsoleted by #1116 :tada: 

Thank you!

**Pull request closed without merging**